File: Front-end/administraccion/administracion/src/core/Entities/pedidos/pedidosEntity.ts
export interface PedidosEntity {
  idOrden: string;
  idProducto: number;
  cantidad: number;
  total: number;
}
  
export   interface PedidoEspacioEntity {
    idOrden: string;
    idProducto: string;
    cantidad: number;
    total: number;
  }
  

File: APP/core/Entities/pedidos/pedidosEntity.py
from pydantic import BaseModel
from typing import Optional

class PedidosEntity(BaseModel):
    idOrden:str
    idProducto:int
    cantidad:int
    total:float

    
class PedidoEspacioEntity(BaseModel):
    idOrden:str
    idProducto:str
    cantidad:float
    total:float

